usr/lib/common/mech_aes.c

```diff
@@ -3636,6 +3992,152 @@ CK_RV ckm_aes_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl)
 }
 
 
+//
+//
+CK_RV ckm_aes_xts_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl)
+{
```

Comment:
This function seem to be a copy of ckm_aes_key_gen with only a few changes. 
I would modify ckm_aes_key_gen and pass a key type parameter to it instead of duplicating all the code